wasm/text/decoder.go

```diff
@@ -585,11 +585,17 @@ func (p *moduleParser) resolveFunctionIndices(module *wasm.Module) error {
 		if err != nil {
 			return err
 		}
+		importCount := p.funcNamespace.count - uint32(len(module.CodeSection))
```

Comment:
can this be calculated outside of loop?

Reply:
indeed. the other TODO, when fixed, will fix this!